ods_ci/tests/Resources/Page/ODH/AiApps/ManagedStarburst.resource

```diff
@@ -14,16 +14,16 @@ ${STARBURST_ROUTE}=    web-ui
 *** Keywords ***
 Create Starburst Route
     [Documentation]    Exposes starburst service for external connections
-    [Arguments]    ${name}
-    ${rc}=    Run And Return Rc    oc expose svc starburst --name ${name} -n ${STARBURST_CR_DEFAULT_NAMESPACE}
+    [Arguments]    ${name}    ${namespace}=${STARBURST_CR_DEFAULT_NAMESPACE}
+    ${rc}=    Run And Return Rc    oc expose svc starburst --name ${name} -n ${namespace}
```

Comment:
I think it's better you validate the return code as soon as the code is run. if something is wrong the and their only you can fail the TC.

I have seen that you are not validating the return code frequently

Reply:
ok, I can add it